itests/deadlines_test.go

```diff
@@ -285,7 +245,11 @@ func TestDeadlineToggling(t *testing.T) {
 	checkMiner(maddrC, types.NewInt(0), true, types.EmptyTSK)
 	checkMiner(maddrB, types.NewInt(uint64(ssz)*sectorsB), true, types.EmptyTSK)
 	checkMiner(maddrD, types.NewInt(uint64(ssz)*sectorsD), true, types.EmptyTSK)
-	checkMiner(maddrE, types.NewInt(0), false, types.EmptyTSK)
+
+	// TODO: Review: minerE has only precommittede some sectors ~2 proving periods
+	//  ago. On older network versions it would become inactive (the bool would be
+	//  false, and was false on older NVs). That doesn't appar to happen now. Why?
```

Comment:
We 30xed the Prove commit delay when moving to [nv13](https://github.com/filecoin-project/go-state-types/blob/master/network/version.go#L23) so 

[MaxProveCommitDelay actors-v4 / nv12](https://github.com/filecoin-project/specs-actors/blob/release/v4/actors/builtin/miner/policy.go#L174)
[MaxProveCommitDelay actors-v5 / nv13](https://github.com/filecoin-project/specs-actors/blob/release/v5/actors/builtin/miner/policy.go#L159)

This is the time it takes precommit to expire.  The hack we use to determine if cron should stay active is examining the locked funds accounting of a miner and determining if it is 0.  So cron won't stop until precommit is expired and with this change that is taking 30x longer